packages/bot-web-ui/src/components/bot-notification/bot-notification.tsx

```diff
@@ -28,19 +36,19 @@ const BotSnackbar = ({ message, timeout = 6000, is_open, onClick, handleClose, t
             onMouseLeave={() => {
                 setNotificationTimer(timeout);
             }}
-            className={className ?? 'bot-snackbar'}
-            data-testid='bot-snackbar-notification-container'
+            className={className ?? 'bot-notification'}
+            data-testid='bot-notification-container'
         >
             <Toast is_open={is_open} type={type} timeout={notification_timer} onClick={onClick} onClose={handleClose}>
                 <div>{message}</div>
                 <Icon
                     icon='IcCross'
-                    className={'notification-close'}
-                    data_testid={'bot-snackbar-notification-close'}
+                    className={'bot-notification__close-icon'}
+                    data_testid={'bot-notification-close'}
```

Comment:
@farabi-deriv same comment as the above data-testid

Reply:
👍